allowances/contracts/AlowanceModule.sol

```diff
@@ -296,11 +301,11 @@ contract AllowanceModule is SignatureDecoder {
     /// @dev Allows to add a delegate.
     /// @param delegate Delegate that should be added.
     function addDelegate(address delegate) public {
-        require(delegate != address(0), "Invalid delegate address");
         uint48 index = uint48(delegate);
+        require(index != uint(0), "index != uint(0)");
         address currentDelegate = delegates[msg.sender][index].delegate;
         if(currentDelegate != address(0)) {
-            // We have a collision for the indeces of delegates 
+            // We have a collision for the indeces of delegates
```

Comment:
indexes

Reply:
indices would be the correct one https://www.yourdictionary.com/indices#:~:text=The%20definition%20of%20indices%20is,noun

Reply:
D'oh :sweat_smile: